src/main/java/it/pagopa/pagopa/apiconfig/controller/IcaController.java

```diff
@@ -59,7 +59,7 @@
             @ApiResponse(responseCode = "500", description = "Service unavailable.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ProblemJson.class)))})
     @GetMapping(
             value = "/{idica}",
-            produces = {"application/xml"}
+            produces = {"application/xml", "application/json"}
```

Comment:
why ?

Reply:
The file is xml, but if an error occurred ProblemJson is returned as JSON.